### ISSUE TITLE
why square root in conf loss mask? 

### ISSUE THREAD

**Issue by sfarkya** [marvis/pytorch-yolo2#164]
Can you please elaborate why there is sqrt on the conf mask is this line for the loss function? [code link](https://github.com/marvis/pytorch-yolo2/blob/6c7c1561b42804f4d50d34e0df220c913711f064/region_loss.py#L162)

Also, in the paper, the authors apply sqrt on w and h. I don't see that in the code. 
Can you please elaborate on that?
